### PR TITLE
Add Weaviate

### PR DESCRIPTION
**Pull request by bobvanluijt**
Adding Weaviate related links

# Requirements for your pull request
- [x] I have read and understood the contribution.MD file

### CODE REVIEW

**Comment by mgramin** [comment]
Hi. Thanks for your PR. how about specifying a link to the github repository directly https://github.com/semi-technologies/weaviate-cli ?

**Comment by bobvanluijt** [comment]
Thanks for the suggestion @mgramin 

@StefanBogdan can you maybe help with this? It might be nice to add some _tools_ around the Weaviate DB?

File: README.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ Feel free to add information about your own db-tools or your favorite third-part
 - [pspg](https://github.com/okbob/pspg) - Postgres Pager.
 - [SQLcl](http://www.oracle.com/technetwork/developer-tools/sqlcl/overview/index.html) - Oracle SQL Developer Command Line (SQLcl) is a free command line interface for Oracle Database.
 - [usql](https://github.com/xo/usql) - A universal command-line interface for PostgreSQL, MySQL, Oracle Database, SQLite3, Microsoft SQL Server, [and many other databases](https://github.com/xo/usql#database-support) including NoSQL and non-relational databases!
+- [Weaviate CLI](https://www.semi.technology/developers/weaviate/current/client-libraries/cli.html) - A CLI tool to interact with a [Weaviate](https://github.com/semi-technologies/weaviate) instance.
```
Comment (StefanBogdan) [inline README.md:100]: This link https://www.semi.technology/developers/weaviate/current/client-libraries/cli.html
Should be this one:  https://github.com/semi-technologies/weaviate-cli

**Comment by StefanBogdan** [comment]
Hi @bobvanluijt , as I see you forked the repo on your privater account, so I think it would be faster if you just change the Weaviate CLI SeMI Technologies page link to the Weaviate CLI GitHub link. I added a comment (see above comment).

I could do it too, if you add me to the repo.

File: README.md [hunk 2]
```diff
@@ -261,6 +262,7 @@ Feel free to add information about your own db-tools or your favorite third-part
 
 ## Schema samples
 - [Oracle Database Sample Schemas](https://github.com/oracle/db-sample-schemas) - Oracle Database Sample Schemas.
+- [Weaviate examples](https://github.com/semi-technologies/weaviate-examples) - A list of schemas and datasets for the [Weaviate](https://github.com/semi-technologies/weaviate) vector search engine.
```
Comment (mgramin) [inline README.md:265]: Note to [contributing.md](https://github.com/mgramin/awesome-db-tools/blob/master/contributing.mdl) please - `Submit only one link at the time`

**Comment by mgramin** [comment]
Thanks for your contribution, but I close this PR for now. Please create new one when ready.

**Comment by bobvanluijt** [comment]
> Thanks for your contribution, but I close this PR for now. Please create new one when ready.

Perfect, thanks

**Pull request closed without merging**